src/Components/Web.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -198,11 +198,13 @@ function createEmscriptenModuleInstance(resourceLoader: WebAssemblyResourceLoade
     MONO.loaded_files = [];
 
     // Fetch the assemblies and PDBs in the background, telling Mono to wait until they are loaded
+    // Mono requires the assembly filenames to have a '.dll' extension, so supply such names regardless
```

Comment:
.NET WebAssembly

Reply:
Sorry, already merged! I doubt this change warrants a further PR so will leave it, but if you disagree let me know and I'll do it.